src/applications/coronavirus-chatbot/utils.js

```diff
@@ -14,6 +22,7 @@ export const watchForButtonClicks = () => {
     const buttons = document.getElementsByClassName('ac-pushButton');
     for (let i = 0; i < buttons.length; i++) {
       buttons[i].addEventListener('click', disableButtons);
+      buttons[i].addEventListener('click', scrollToNewMessage);
```

Comment:
This is confusing - two event listeners applied to each element in a dynamic list of elements...I'm wondering if we can reduce this to -

```js
// When the webchat is starting up - this is attached to the root element and only attached once
root.addEventListener('click', event => {
    const node = event.target;
    const isButton = node.classlist.includes('ac-pushButton');
    if (!isButton) return;
    
    disableButtons();
    scrollToNewMessage();
});
```

Reply:
This PR was merged quickly, but we looked into this approach afterwards:
The event listener on each button (added by botframework-webchat) doesn't bubble the event up so our event listener on root was never getting executed. It seems messier to change the webchat code, so we stuck with applying our event listener dynamically.